pkg/client/events.go

```diff
@@ -172,3 +168,29 @@ func (e *events) Delete(name string) error {
 		Do().
 		Error()
 }
+
+func (e *events) GetFieldSelector(involvedObjectName, involvedObjectNamespace, involvedObjectKind, involvedObjectUID *string) labels.Selector {
```

Comment:
Done.
But isnt it duplication of comment now? The same comment is duplicated at 2 places.